src/applications/personalization/dashboard/tests/e2e/benefit-applications-submitted.cypress.spec.js

```diff
@@ -142,4 +143,71 @@ describe('The My VA Dashboard', () => {
       cy.axeCheck();
     });
   });
+
+  describe('a submitted form is received and has a PDF download link', () => {
+    const filename = 'Form_21-10210_123fake-submission-id-567.pdf';
+
+    beforeEach(() => {
+      cy.intercept('GET', '/v0/feature_toggles*', {
+        data: {
+          features: [
+            {
+              name: featureFlagNames.myVaFormPdfLink,
+              value: true,
+            },
+          ],
+        },
+      });
+      cy.intercept('/v0/my_va/submission_statuses', formStatusesNoError);
+      cy.intercept('POST', '/v0/my_va/submission_pdf_urls', {
```

Comment:
hrmm, this doesn't seem like the right behavior to me.  The `/v0/my_va/submission_pdf_urls` url should return json with the `url` to the pdf file.  Once the download button is clicked, another request will be made to download the file.

I think we'll need to split this into 2 interceptors, one for `/v0/my_va/submission_pdf_urls`, and another for `*.pdf`.